zeppelin-web/src/app/app.js

```diff
@@ -109,11 +108,18 @@
   }
 
   function bootstrapApplication() {
+    zeppelinWebApp.run(function($rootScope, $location) {
+      $rootScope.$on('$routeChangeStart', function(event, next, current) {
+        if (!$rootScope.ticket && !next.$$route.publicAccess) {
+          event.preventDefault();
```

Comment:
`event.preventDefault();` is not letting my page to render. On removing this works as expected.